streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/JacksonUtils.java

```diff
@@ -38,7 +38,7 @@ private JacksonUtils() {}
   static {
     MAPPER = new ObjectMapper();
     MAPPER.registerModule(new DefaultScalaModule());
-    MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true);
```

Comment:
This method is used in many places, and changing this parameter could potentially cause other bugs. Unless it has undergone thorough testing, can you explain why it is necessary to change this parameter?